chcemvediet/apps/inforequests/sitemaps.py

```diff
@@ -0,0 +1,31 @@
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.contrib.sitemaps import Sitemap
+from django.db.models import Max
+
+from poleno.utils.urls import reverse
+from poleno.utils.translation import translation
+
+from .models import Action, Inforequest
+
+
+class InforequestsSitemap(Sitemap):
+    changefreq = u'weekly'
+    priority = 0.5
+
+    def items(self):
+        res = []
+        inforequests = Inforequest.objects.published()
+        for lang, name in settings.LANGUAGES:
+            for inforequest in inforequests:
+                res.append((lang, inforequest))
+        return res
+
+    def location(self, (lang, inforequest)):
+        with translation(lang):
+            return reverse(u'inforequests:detail', args=[inforequest.slug, inforequest.pk])
+
+    def lastmod(self, (lang, inforequest)):
+        return (Action.objects
+                .where_inforequest(inforequest)
+                .aggregate(most_recent=Max(u'created'))[u'most_recent'])
```

Comment:
Imho namiesto `.aggregate(most_recent=Max(u'created'))` by bolo efektivnejsie selectnut najnovsiu akciu a zobrat jej `created` cas. Analogicky ako `newest` metoda v `InforequestEmailQuerySet`. Aj potom pouzitie by bolo zrozumitelnejsie: `Action.objects.of_inforequest(inforequest).newest()`

Reply:
V tom pripade by sme mohli pouzit aj metodu QuerySet-u `latest()`, nemuseli by sme zbytocne definovat dalsiu. Ale metoda latest (a aj newest) vyuziva order_by, a sortovanie ma vacsiu zlozitost ako aggregate. Potvrdilo sa mi to aj pri testovani vsetkych troch moznosti, kde som meral cas, najlepsi cas mal `aggregate`, potom `latest` a  potom `newest()`.

Reply:
Ok. Tak teda nechajme agregaciu.